### PR TITLE
export stats_persist_period_sec

### PR DESCRIPTION
**Pull request by developerfred**
fixed
 - https://github.com/rust-rocksdb/rust-rocksdb/issues/447 
 -  https://github.com/rust-rocksdb/rust-rocksdb/pull/448

### CODE REVIEW

**Comment by DarkEld3r** [comment]
You should add an implementation for these functions in the `db/c.cc` file and probably tests in the `db/c_test.c` file.

**Comment by developerfred** [comment]
> You should add an implementation for these functions in the `db/c.cc` file and probably tests in the `db/c_test.c` file.

@DarkEld3r Thanks I will do!

File: db/c_test.c [hunk 1]
```diff
@@ -1825,6 +1828,7 @@ int main(int argc, char** argv) {
     CheckCondition(1 == rocksdb_options_get_is_fd_close_on_exec(copy));
     CheckCondition(1 == rocksdb_options_get_skip_log_error_on_recovery(copy));
     CheckCondition(18 == rocksdb_options_get_stats_dump_period_sec(copy));
+    CheckCondition(600 == rocksdb_options_get_stats_persist_period_sec(copy));
```
Comment (DarkEld3r) [inline db/c_test.c:1831]: I suppose there should be 5 here, not 600.

Comment (developerfred) [reply db/c_test.c]: done

**Pull request closed without merging**